DeckBuilder/Include/Deck.hpp

```diff
@@ -1,24 +1,25 @@
 #pragma once
 #include <string>
 #include <vector>
-#include "Card.hpp"
+#include "../../db/card_api/Include/baseCardApi.hpp"
```

Comment:
jak jest cmake dobrze zrobiony, to nie trzeba podawać całej scieżki z ../../ itd.